### DISCUSSION TITLE
Are the docker images not public?

### DISCUSSION THREAD

**Discussion by andrejohansson** [cross-rs/cross#691] [Q&A]
I'm trying to use the docker images in a build, but the registry seems to require authentication:

```
❯ docker pull ghcr.io/cross-rs/x86_64-apple-darwin:latest
Error response from daemon: Head "https://ghcr.io/v2/cross-rs/x86_64-apple-darwin/manifests/latest": denied
```

So, is this not possible to do? Do I just authenticate using a PAT from my github account (which permissions)?

**Comment by Emilgardis** [comment]
We do not have the latest tag published yet on ghcr, when we do the 0.3.0 release, it'll be available on all images as `latest` and `0.3.0`. Use the `main` tag for now.

However! We don't have a darwin image, see https://github.com/cross-rs/cross/issues/223

**Reply by Emilgardis** [reply]
You can see all images/packages here https://github.com/orgs/cross-rs/packages?repo_name=cross

**Reply by andrejohansson** [reply]
ok, thank you for your answer! 

Hope there will be a darwin image soon, would be really nice :-)
